Example/Auth/Sample/MainViewController.m

```diff
@@ -1776,12 +1790,14 @@ - (void)signinWithProvider:(id<AuthProvider>)authProvider retrieveData:(BOOL)ret
           [self logFailure:@"sign-in with provider failed" error:error];
         } else {
           [self logSuccess:@"sign-in with provider succeeded."];
-          NSString *newUserString = authResult.additionalUserInfo.isNewUser ?
-          @"new user" : @"existing user";
-          [self showMessagePromptWithTitle:@"New Or Exisiting"
-                                   message:newUserString
-                          showCancelButton:NO
-                                completion:nil];
+          if (_isNewUserToggleOn) {
```

Comment:
This code should be inside the "if (authResult.additionalUserInfo) {" condition.  It seems like you didn't pull in the changes that you merged into master before staring this CL. you can fix this by pulling the changes into you local master and rebasing this branch on you updates master.